File: Sduty_Server/src/main/java/com/d108/sduty/dto/Task.java
package com.d108.sduty.dto;

import io.swagger.annotations.ApiModel;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@ApiModel(value = "Task: report's task 정보", description = "리포트의 task 상세정보")
public class Task {
	private int seq;
	private String title;
	private String content;
	private String startTime;
	private String endTime;
	private String durationTime;
	
}

File: Sduty_User/app/src/main/java/com/d108/sduty/model/dto/Task.kt
package com.d108.sduty.model.dto

import android.os.Parcel
import android.os.Parcelable
import kotlinx.parcelize.Parcelize

@Parcelize
data class Task(
    val seq: Int,
    val reportSeq: Int,
    val endTime: String,
    val startTime: String,
    val durationTime: Int,
    val title: String,
    val content1: String,
    val content2: String,
    val content3: String
) : Parcelable {
    override fun describeContents(): Int {
        return 0
    }

    override fun writeToParcel(dest: Parcel?, flags: Int) {}
}